### DISCUSSION TITLE
官方社区账号找回密码失败

### DISCUSSION THREAD

**Discussion by sibyl2116** [1Panel-dev/1Panel#4286] [Q&A]
点击后邮箱收不到任何邮件，收信记录中没有

![Clip_20240323_160810](https://github.com/1Panel-dev/1Panel/assets/102795484/308a6fc4-1836-43e9-8680-b1057a670478)

**Comment by wanghe-fit2cloud** [comment]
提示：注册的时候是手机号加验证码注册的，登录也直接验证码登录即可。

**Reply by sibyl2116** [reply]
已找回
